app/views/admin/role_appointments/edit.html.erb

```diff
@@ -1,40 +1,74 @@
-<% page_title "Edit appointment to #{@role_appointment.role.name}"  %>
+<% content_for :page_title, "Edit appointment to #{@role_appointment.role.name}"  %>
+<% content_for :title, "Edit appointment to #{@role_appointment.role.name}" %>
+<% content_for :title_margin_bottom, 6 %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @role_appointment, parent_class: "role_appointment")) %>
 
-<div class="row">
-  <div class="col-md-8">
-    <h1>Edit appointment to position of <%= @role_appointment.role.name %> </h1>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_for [:admin, @role_appointment] do |form| %>
+      <%= render 'admin/role_appointments/form', form: form %>
 
-    <%= form_for [:admin, @role_appointment], html: {class: 'well'} do |form| %>
-      <%= render partial: 'admin/role_appointments/form', locals: { form: form } %>
-
-      <%= form.save_or_cancel cancel: edit_admin_role_path(@role_appointment.role) %>
+      <div class="govuk-button-group">
+        <%= render "govuk_publishing_components/components/button", {
+          text: "Save",
+          name: "save",
+          value: "Save",
+          type: "submit",
+          data_attributes: {
+            module: "gem-track-click",
+            "track-category": "form-button",
+            "track-action": "#{form.object.class.name.demodulize.underscore.dasherize}-button",
+            "track-label": "Save"
+          }
+        } %>
+        <%= link_to("Cancel", edit_admin_role_path(@role_appointment.role), class: "govuk-link") %>
+      </div>
     <% end %>
 
     <% if @role_appointment.destroyable? %>
-      <%= button_to "Delete", admin_role_appointment_path(@role_appointment), method: :delete, class: "btn btn-danger" %>
+      <%= form_for [:admin, @role_appointment], method: "delete" do |form| %>
```

Comment:
Sorry we should put this in the docs but we should add confirm delete pages for destroying objects. 

To follow the established pattern i 'd think this should have a red delete link inline between the save button and cancel link

Reply:
Thanks, yup, have updated to reflect this.

Reply:
Thanks! Sorry we should add this to the card when required.